Makefile

```diff
@@ -0,0 +1,13 @@
+all: app editor web
+
+app:
+	docker build -f ./app/Dockerfile.base -t driver/app:latest ./app
+	docker build -f ./app/Dockerfile.development -t driver/app-dev:latest ./app
+
+editor:
+	docker build -f ./schema_editor/Dockerfile -t driver/editor:latest ./schema_editor
+
+web:
+	docker build -f ./web/Dockerfile -t driver/web:latest ./web
+
+.PHONY: all app editor web
```

Comment:
It does not. I originally set it up to help get things rolling using a Docker client on my VM host (after overriding `DOCKER_HOST=tcp://127.0.0.1:2375` it should work).

Do you think it would be handy to keep and document in the `README` or just delete it?